README.md

```diff
@@ -169,11 +169,10 @@ If a `save-artifacts` script exists, a prior image already exists, and the `--in
 
 ## Dependencies
 
-1. [Docker](http://www.docker.io) >= 1.6
-1. [Go](http://golang.org/) >= 1.4
+1. [docker](https://github.com/docker/docker) >= 1.6
```

Comment:
Now referring to the GitHub repository instead of Docker Engine. Should we bump the minimum docker version as well?